va/va_drmcommon.h

```diff
@@ -155,4 +160,23 @@ typedef struct _VADRMPRIMESurfaceDescriptor {
 } VADRMPRIMESurfaceDescriptor;
 
 
+typedef struct _VADRMPRIMESurfaceDescriptorExt {
+    VADRMPRIMESurfaceDescriptor surface_desc;
+    /* memory location , local memory or system memory */
+    union{
+        struct{
+            /* whether the memory is allocated on local memory */
+            uint32_t local_memory :1;
```

Comment:
What does "local memory" mean?  Is it memory local to the CPU running the user code, memory local to the device being used, something else?

Reply:
memory local to the device being used. it is on chip/device memory.